meta-lmp-bsp/recipes-security/optee/optee-os-fio_3.17.0.bbappend

```diff
@@ -1 +1,20 @@
 require optee-os-fio-bsp.inc
+
+OPTEE_SUFFIX:stm32mp1common    = "bin"
+OPTEE_HEADER:stm32mp1common    = "tee-header_v2"
+OPTEE_PAGEABLE:stm32mp1common  = "tee-pageable_v2"
+OPTEE_PAGER:stm32mp1common     = "tee-pager_v2"
+
+do_deploy:append:stm32mp1common() {
+    if [ -n "${OPTEE_CONF}" ]; then
+        for conf in ${OPTEE_CONF}; do
+            install -m 644 ${B}/core/${OPTEE_HEADER}.${OPTEE_SUFFIX} ${DEPLOYDIR}/optee/${OPTEE_HEADER}-${conf}.${OPTEE_SUFFIX}
+            install -m 644 ${B}/core/${OPTEE_PAGER}.${OPTEE_SUFFIX} ${DEPLOYDIR}/optee/${OPTEE_PAGER}-${conf}.${OPTEE_SUFFIX}
+            install -m 644 ${B}/core/${OPTEE_PAGEABLE}.${OPTEE_SUFFIX} ${DEPLOYDIR}/optee/${OPTEE_PAGEABLE}-${conf}.${OPTEE_SUFFIX}
+        done
+    else
+        install -m 644 ${B}/core/${OPTEE_HEADER}.${OPTEE_SUFFIX} ${DEPLOYDIR}/optee/${OPTEE_HEADER}.${OPTEE_SUFFIX}
+        install -m 644 ${B}/core/${OPTEE_PAGER}.${OPTEE_SUFFIX} ${DEPLOYDIR}/optee/${OPTEE_PAGER}.${OPTEE_SUFFIX}
+        install -m 644 ${B}/core/${OPTEE_PAGEABLE}.${OPTEE_SUFFIX} ${DEPLOYDIR}/optee/${OPTEE_PAGEABLE}.${OPTEE_SUFFIX}
```

Comment:
e.g. from kria (what I have locally):
```
rsalveti@evatuf:~/build/lmp/build-lmp$ ls -l deploy/images/kv260/optee/
total 4684
-rw-r--r-- 1 rsalveti rsalveti  416100 Jun  8 13:42 tee.bin
-rw-r--r-- 1 rsalveti rsalveti 4012464 Jun  8 13:42 tee.elf
-rw-r--r-- 1 rsalveti rsalveti      28 Jun  8 13:42 tee-header_v2.bin
-rw-r--r-- 1 rsalveti rsalveti      11 Jun  8 13:42 tee-init_load_addr.txt
-rw-r--r-- 1 rsalveti rsalveti       0 Jun  8 13:42 tee-pageable_v2.bin
-rw-r--r-- 1 rsalveti rsalveti  416072 Jun  8 13:42 tee-pager_v2.bin
```

So I think we are safe to just keep the if block.